app/experimenter/experiments/api/v5/types.py

```diff
@@ -37,3 +38,26 @@ class NimbusBucketRangeType(DjangoObjectType):
     class Meta:
         model = NimbusBucketRange
         exclude = ("id", "experiment")
+
+
+class NimbusExperimentIdOrder(Enum):
+    ASCENDING = "ascending"
+    DESCENDING = "descending"
+
+
+class NimbusExperimentOptionalStatus(Enum):
+    """Nimbus Experiment Optional Status
+
+    The NimbusExperimentStatus type on the NimbusExperimentType is a required
+    field preventing it from being used as an optional argument. This
+    duplicate is used to allow for optional usage as a Query argument.
+
+    """
+
+    DRAFT = "Draft"
+    REVIEW = "Review"
+    SHIP = "Ship"
+    ACCEPTED = "Accepted"
+    LIVE = "Live"
+    COMPLETE = "Complete"
+    REJECTED = "Rejected"
```

Comment:
Can't we just refer to the constants here? https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/constants/nimbus.py#L5-L12

Reply:
I was not having any luck attempting to rope those in, I tried about a dozen various invocations and failed before giving up and just putting this together so it would work.

Reply:
Looking at:

https://docs.graphene-python.org/en/latest/types/enums/#definition

Seems like this works:

```py
NimbusExperimentOptionalStatus = Enum(
    "NimbusExperimentOptionalStatus", NimbusConstants.Status.choices
)
```

Reply:
I know, it really seems like it should work, right? Unfortunately it doesn't.

Reply:
I just tried it and the tests passed? Where doesn't it work?

Reply:
```
diff --git a/app/experimenter/experiments/api/v5/types.py b/app/experimenter/experiments/api/v5/types.py
index a0093659..f008b139 100644
--- a/app/experimenter/experiments/api/v5/types.py
+++ b/app/experimenter/experiments/api/v5/types.py
@@ -8,6 +8,7 @@ from experimenter.experiments.models.nimbus import (
     NimbusIsolationGroup,
 )
 from experimenter.projects.models import Project
+from experimenter.experiments.constants import NimbusConstants
 
 
 class NimbusBranchType(DjangoObjectType):
@@ -45,19 +46,6 @@ class NimbusExperimentIdOrder(Enum):
     DESCENDING = "descending"
 
 
-class NimbusExperimentOptionalStatus(Enum):
-    """Nimbus Experiment Optional Status
-
-    The NimbusExperimentStatus type on the NimbusExperimentType is a required
-    field preventing it from being used as an optional argument. This
-    duplicate is used to allow for optional usage as a Query argument.
-
-    """
-
-    DRAFT = "Draft"
-    REVIEW = "Review"
-    SHIP = "Ship"
-    ACCEPTED = "Accepted"
-    LIVE = "Live"
-    COMPLETE = "Complete"
-    REJECTED = "Rejected"
+NimbusExperimentOptionalStatus = Enum(
+    "NimbusExperimentOptionalStatus", NimbusConstants.Status.choices
+)
diff --git a/app/experimenter/experiments/tests/api/v5/test_query.py b/app/experimenter/experiments/tests/api/v5/test_query.py
index b16ce90d..7a538391 100644
--- a/app/experimenter/experiments/tests/api/v5/test_query.py
+++ b/app/experimenter/experiments/tests/api/v5/test_query.py
@@ -45,7 +45,7 @@ class TestExperimentListView(GraphQLTestCase):
         response = self.query(
             """
             query {
-                experiments(status: DRAFT) {
+                experiments(status: Draft) {
                     name
                     slug
                     publicDescription
```

Reply:
`
    experiments = graphene.List(
        NimbusExperimentType,
        description="List Nimbus Experiments.",
        offset=graphene.Int(),
        limit=graphene.Int(),
        order=NimbusExperimentIdOrder(),
        status=graphene.Enum(
            "NimbusExperimentOptionalStatus", NimbusConstants.Status.choices
        ),
    )
`

It's not working here.

Reply:
Ugh, the traceback is annoyingly devoid of helpful advice. I was missing an additional instantiation.